cloud-controller-manager/osc/utils.go

```diff
@@ -476,26 +558,26 @@ func isRegionValid(region string, metadata EC2Metadata) bool {
 }
 
 // newAWSInstance creates a new awsInstance object
-func newAWSInstance(ec2Service Compute, instance *ec2.Instance) *VM {
+func newAWSInstance(ec2Service Compute, instance *osc.Vm) *VM {
 	az := ""
 	if instance.Placement != nil {
-		az = aws.StringValue(instance.Placement.AvailabilityZone)
+		az = instance.Placement.GetSubregionName()
 	}
 	self := &VM{
 		compute:          ec2Service,
-		awsID:            aws.StringValue(instance.InstanceId),
+		awsID:            aws.StringValue(instance.VmId),
 		nodeName:         mapInstanceToNodeName(instance),
 		availabilityZone: az,
-		instanceType:     aws.StringValue(instance.InstanceType),
-		vpcID:            aws.StringValue(instance.VpcId),
+		instanceType:     aws.StringValue(instance.VmType),
```

Comment:
aws.StringValue can be changed or not ?

Reply:
will remove the use of this function, thank you 👍